File: C#/Book_Store_C#/CSD221_Lab6_Todd_Pidgeon/Lab6/Cashtill.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab6
{
    /**
    Author : Todd Pidgeon
    Date : 03/06/2017
    Description : the class structure for CashTill.
     **/

    class CashTill
    {
        private Double runningTotal;
        public void sellItem(SaleableItem p)
        {
            runningTotal += p.getPrice();
            p.sellCopy();
        }
        public CashTill()
        {
            runningTotal = 0;
        }
    }
}

File: C#/Book_Store_C#/Lab4/CashTill.java

package Lab4;

/**
Author : Todd Pidgeon
Date : 03/06/2017
Description : the class structure for CashTill.
**/

public class CashTill {
    private double runningTotal;
    
    CashTill(){
        runningTotal = 0;
    }
    
    public void sellItem(SaleableItem pPub){
        runningTotal = runningTotal + pPub.getPrice();
        pPub.sellCopy();
        System.out.println("\nSold\n" + pPub + " @ " + pPub.getPrice() 
                + "\nSubtotal = " + runningTotal);
    }
    
    public void showTotal(){
        System.out.println("Grand Total: " + runningTotal);
    }
}
